locale/de/LC_MESSAGES/pronterface.po

```diff
@@ -440,947 +507,776 @@ msgid ""
-msgstr ""
+msgstr "Wählen Sie die Datei zum Speichern aus"
 
-#: printrun/plater.py:367
-msgid ""
-"Failed to use simarrange for plating, falling back to the standard method"
+#: printrun/objectplater.py:288
+msgid "Plate building tool"
 msgstr ""
 
-#: printrun/plater.py:372
-msgid "Autoplating using simarrange"
+#: printrun/printcore.py:235
+msgid "Could not connect to %s:%s:"
 msgstr ""
 
-#: printrun/plater.py:387
-msgid "Plate full, please remove some objects"
+#: printrun/printcore.py:236
+msgid "Socket error %s:"
 msgstr ""
 
-#: printrun/plater.py:404
-msgid "simarrange failed"
+#: printrun/printcore.py:264 printrun/printcore.py:269
+msgid "Could not connect to %s at baudrate %s:"
 msgstr ""
 
-#: printrun/printcore.py:177
-msgid "Could not connect to %s:%s:"
+#: printrun/printcore.py:265
+msgid "Serial error: %s"
 msgstr ""
 
-#: printrun/printcore.py:178
-msgid "Socket error %s:"
+#: printrun/printcore.py:270
+msgid "IO error: %s"
 msgstr ""
 
-#: printrun/printcore.py:191 printrun/printcore.py:196
-msgid "Could not connect to %s at baudrate %s:"
+#: printrun/printcore.py:335
+msgid "Can't read from printer (disconnected?). line_bytes is None"
 msgstr ""
 
-#: printrun/printcore.py:192
-msgid "Serial error: %s"
+#: printrun/printcore.py:350
+msgid "Got rubbish reply from %s at baudrate %s:"
 msgstr ""
 
-#: printrun/printcore.py:197
-msgid "IO error: %s"
+#: printrun/printcore.py:351
+msgid "Maybe a bad baudrate?"
 msgstr ""
 
-#: printrun/printcore.py:231
+#: printrun/printcore.py:355
 msgid "Can't read from printer (disconnected?) (SelectError {0}): {1}"
 msgstr ""
 
-#: printrun/printcore.py:234
+#: printrun/printcore.py:358
 msgid "SelectError ({0}): {1}"
 msgstr ""
 
-#: printrun/printcore.py:237
+#: printrun/printcore.py:361
 msgid "Can't read from printer (disconnected?) (SerialException): {0}"
 msgstr ""
 
-#: printrun/printcore.py:240
+#: printrun/printcore.py:364
 msgid "Can't read from printer (disconnected?) (Socket error {0}): {1}"
 msgstr ""
 
-#: printrun/printcore.py:245
+#: printrun/printcore.py:369
 msgid "Can't read from printer (disconnected?) (OS Error {0}): {1}"
 msgstr ""
 
-#: printrun/printcore.py:259
+#: printrun/printcore.py:383
 msgid "Aborting connection attempt after 4 failed writes."
 msgstr ""
 
-#: printrun/printcore.py:462 printrun/printcore.py:470
-#: printrun/pronsole.py:1192 printrun/pronsole.py:1250
-#: printrun/pronterface.py:209 printrun/pronterface.py:962
-#: printrun/pronterface.py:1111 printrun/pronterface.py:1178
+#: printrun/printcore.py:589 printrun/printcore.py:597
+#: printrun/pronsole.py:1043 printrun/pronsole.py:1101
+#: printrun/pronterface.py:223 printrun/pronterface.py:1161
+#: printrun/pronterface.py:1297 printrun/pronterface.py:1370
 msgid "Not connected to printer."
 msgstr "Keine Verbindung zum Drucker."
 
-#: printrun/printcore.py:479
+#: printrun/printcore.py:609
 msgid "Print start callback failed with:"
-msgstr ""
+msgstr "Druckstart-Rückmeldung fehlgeschlagen mit:"
```

Comment:
I'd use "Rückruffunktion" or "Rückruf" instead of "Rückmeldung" for "callback". I don't think it's clear what it means from the translation. Perhaps even use "Callback". This is referring to a user-provided function that is called when certain events happen.